packages/trader/src/Modules/Trading/Containers/purchase.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { isAccumulatorContract, isEmptyObject, isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import PurchaseButtonsOverlay from 'Modules/Trading/Components/Elements/purchase-buttons-overlay.jsx';
-import PurchaseFieldset from 'Modules/Trading/Components/Elements/purchase-fieldset.jsx';
+import PurchaseFieldset from 'Modules/Trading/Components/Elements/purchase-fieldset';
 import { getContractTypePosition } from 'Constants/contract';
 import { useTraderStore } from 'Stores/useTraderStores';
 import { observer, useStore } from '@deriv/stores';
+import { TProposalTypeInfo } from 'Types';
 import ContractInfo from 'Modules/Trading/Components/Form/Purchase/contract-info.jsx';
 
-const Purchase = observer(({ is_market_closed }) => {
+// @ts-expect-error returned value should be wraped with React.Fragmant (it's an array with components), but we can't do this as it causes issues.
```

Comment:
@kate-deriv let's remove it. assertion that I suggested (return components as unknown as JSX.Element;) will be better than // @ts-expect-error 🙂